connector/src/main/java/org/geysermc/connector/network/UpstreamPacketHandler.java

```diff
@@ -68,6 +71,10 @@ public boolean handle(LoginPacket loginPacket) {
 
         session.getUpstream().getSession().setPacketCodec(packetCodec);
 
+        // Set the block translation based off of version
+        session.setBlockTranslator(packetCodec.getProtocolVersion() >= Bedrock_v428.V428_CODEC.getProtocolVersion()
+                ? BlockTranslator1_16_210.INSTANCE : BlockTranslator1_16_100.INSTANCE);
```

Comment:
I feel like if we expand this it should be implemented as part of the BedrockProtocol class. But that's out of scope for this PR.